pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -462,10 +462,17 @@ func (r *ReconcilePerconaServerMongoDB) reconcileStatefulSet(arbiter bool, cr *a
 	if err != nil {
 		return nil, fmt.Errorf("create StatefulSet.Spec %s: %v", sfs.Name, err)
 	}
-	sfsSpec.Template.Annotations = sfs.Spec.Template.Annotations
+
 	if sfsSpec.Template.Annotations == nil {
 		sfsSpec.Template.Annotations = make(map[string]string)
 	}
+	sfs.Annotations = make(map[string]string)
```

Comment:
maybe its better to check for nil before make()

Reply:
sure

Reply:
@nonemax done